func.js

```diff
@@ -0,0 +1,549 @@
+MATCH (n1:Reaction:${model} {id: csvLine.reactionId}),(n2:CompartmentalizedMetabolite:${model} {id: csvLine.compartmentalizedMetaboliteId})
+CREATE (n1)-[:${version} {stoichiometry:toFloat(csvLine.stoichiometry)}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.reactionGenes.csv" AS csvLine
+MATCH (n1:Reaction:${model} {id: csvLine.reactionId}),(n2:Gene:${model} {id: csvLine.geneId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.reactionSubsystems.csv" AS csvLine
+MATCH (n1:Reaction:${model} {id: csvLine.reactionId}),(n2:Subsystem:${model} {id: csvLine.subsystemId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.reactionPubmedReferences.csv" AS csvLine
+MATCH (n1:Reaction:${model} {id: csvLine.reactionId}),(n2:PubmedReference {id: csvLine.pubmedReferenceId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.compartmentSvgMaps.csv" AS csvLine
+MATCH (n1:Compartment:${model} {id: csvLine.compartmentId}),(n2:SvgMap:${model} {id: csvLine.svgMapId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.subsystemSvgMaps.csv" AS csvLine
+MATCH (n1:Subsystem:${model} {id: csvLine.subsystemId}),(n2:SvgMap:${model} {id: csvLine.svgMapId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.compartmentalizedMetaboliteExternalDbs.csv" AS csvLine
+MATCH (n1:CompartmentalizedMetabolite:${model} {id: csvLine.compartmentalizedMetaboliteId}),(n2:ExternalDb {id: csvLine.externalDbId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.subsystemExternalDbs.csv" AS csvLine
+MATCH (n1:Subsystem:${model} {id: csvLine.subsystemId}),(n2:ExternalDb {id: csvLine.externalDbId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.reactionExternalDbs.csv" AS csvLine
+MATCH (n1:Reaction:${model} {id: csvLine.reactionId}),(n2:ExternalDb {id: csvLine.externalDbId})
+CREATE (n1)-[:${version}]->(n2);
+
+LOAD CSV WITH HEADERS FROM "file:///${prefix}.geneExternalDbs.csv" AS csvLine
+MATCH (n1:Gene:${model} {id: csvLine.geneId}),(n2:ExternalDb {id: csvLine.externalDbId})
+CREATE (n1)-[:${version}]->(n2);
+
+`
+  cypherInstructions.split('\n').forEach(i => {
+    instructions.push(i);
+  });
+  return instructions;
+}
+
+const getRemainCyperInstructions = (instructions) => {
+// Get the remaining cyper instructions
+  `CALL db.index.fulltext.createNodeIndex(
+    "fulltext",
+    ["CompartmentState", "Compartment", "MetaboliteState", "Metabolite", "CompartmentalizedMetabolite", "SubsystemState", "Subsystem", "ReactionState", "Reaction", "GeneState", "Gene", "PubmedReference"],
+    ["id", "name", "letterCode", "alternateName", "synonyms", "description", "formula", "function", "pubMedID", "ec"]);
+  `.split('\n').forEach(i => {
+    instructions.push(i);
+  });
+  return instructions;
+}
+
+exports.getFile = getFile;
```

Comment:
Could these exports be done with a similar syntax as in `var.js`? For example:

```javascript
module.exports = {
  getFile,
  toLabelCase,
  ...
}
```

Reply:
module.exports does not work for arrow functions as I tried. Maybe I can use the same syntax for exporting the dictionary in var.js as well. In addition, I think it is probably better to name the file `var.js` as it is in case we will later on add other pre-defined variables to this file.

Reply:
Thanks for updating the files and improving the organization.

I tried having the following in `parser.js` and it seems to work. Does this not work for you?

```javascript
//exports.getInfoFromYaml = getInfoFromYaml;
//exports.getComponentSvgRel = getComponentSvgRel;
//exports.getPMIDs = getPMIDs;
//exports.getGeneAnnotation = getGeneAnnotation;
//exports.getComponentExternalDb = getComponentExternalDb;

module.exports = {
  getInfoFromYaml,
  getComponentSvgRel,
  getPMIDs,
  getGeneAnnotation,
  getComponentExternalDb,
}
```

Reply:
interesting! It was not working before but works now. I guess it is because I mixed the `function` definition and `arrow function` definition previously. I can convert export method to `module.exports`.

Reply:
Nice that you could test and update it so fast, thanks! I was worried that it was due to different versions of node.js but glad that it worked for you.